pkgs/development/ocaml-modules/reason-react/ppx.nix

```diff
@@ -0,0 +1,32 @@
+{
+  buildDunePackage,
+  fetchurl,
+  lib,
+  ppxlib,
+}:
+
+let
+  version = "0.15.0";
+  minimalOCamlVersion = "4.14";
+in
```

Comment:
Why are these defined here when they are just inherited later on?

Reply:
I'll move `minimalOCamlVersion` down, this was just a left-over from the previous reusage.

Reply:
Regarding `version`, however, that one is not just inherited, but also used in the source `url`.
(and I read in the Nix coding style that `let ... in` should be preferred over `rec {`).